src/mlpack/bindings/markdown/binding_info.hpp

```diff
@@ -25,15 +30,12 @@ namespace markdown {
 class BindingInfo
 {
  public:
-  /**
-   * Return a ProgramDoc object for a given bindingName.
-   */
+  
+  //! Return a ProgramDoc object for a given bindingName.
   static util::ProgramDoc& GetProgramDoc(const std::string& bindingName);
 
-  /**
-   * Register a ProgramDoc object with the given bindingName.
-   */
-  static void RegisterProgramDoc(const std::string& bindingName,
+   //! Register a ProgramDoc object with the given bindingName.
+   static void RegisterProgramDoc(const std::string& bindingName,
                                  const util::ProgramDoc& programDoc);
```

Comment:
Looks like an extra space was added here; that's probably what the style issue is.

Reply:
Since it is the not a file Doc comment, I think we do not need a Document Comment Style '*/' . That's what I corrected.

Reply:
Agreed, and that's just fine, but now it's indented 3 spaces instead of 2, that's the issue. :)

Reply:
Yes sir. I will correct it. Sorry. Also I have finished documentation of 'Markdown' folder which you can check for errors.

Reply:
@rcurtin I have finished the documentation of this folder.
Can you help me for clearing some of the things for Java Bindings?

Reply:
Thanks, looks good now. :+1:  I'm not sure what you mean about Java bindings, can you clarify?